alerts.md

```diff
@@ -2,42 +2,64 @@
 
 ### Summary
 
-Kubecost alerts allow teams to receive updates on real-time Kubernetes spend. They are configurable via the Kubecost UI, and now via Helm values. This resource gives an overview of how to configure Kubecost email and Slack alerts using [Kubecost helm chart values](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml).
+Kubecost alerts allow teams to receive updates on real-time Kubernetes spend. They are configurable via Helm values. This resource gives an overview of how to configure Kubecost email and Slack alerts using [Kubecost helm chart values](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml).
   
 
-Kubecost currently supports four different types of alerts:
-
- 1. Recurring update, weekly (all namespaces)
-	 - Sends an email to the configured global email(s) reporting cluster spend across all namespaces, broken down by namespace
-
- 2. Recurring update, weekly by namespace
-	 - Sends an email to the configured owner contact email, defaulting to global emails if not provided, reporting individual namespace spend, identified by `filter`
-
- 3. Daily budget alert by cluster
-
-	 - Sends an email and Slack alert to global email(s) and Slack webhook (if configured), reporting daily total cluster spend
-
- 4. Daily budget alert by namespace
-
-	 - Sends an email and Slack alert to the configured owner contact, defaulting to global emails if not provided, reporting daily total namespace spend, identified by `filter`
+As of v1.72.0, Kubecost supports three types of alerts:
+
+ 1. Recurring update
+   
+   - Weekly (all namespaces) -- sends an email and Slack alert reporting cluster spend across all namespaces, with costs broken down by namespace
+     - Required parameters:
+       - `type: recurringUpdate`
+       - `aggregation: namespace`
+       - `filter: '*'`
+       - `window: 7d`
+   - Weekly (by namespace) -- sends an email and Slack alert reporting individual namespace spend, identified by `filter`
+     - Required parameters:
+       - `type: recurringUpdate`
+       - `aggregation: namespace`
+       - `filter: <value>` -- configurable, accepts a single namespace name (comma separated values unsupported)
+       - `window: 7d`
+
+ 2. Budget -- sends an email and Slack alert reporting spend
+     - Required parameters:
+       - `type: budget`
+       - `threshold: <amount>` -- configurable, cost threshold in configured currency units
+       - `aggregation: <agg-parameter>` -- configurable, accepts all aggregations supported by the [aggregated cost model API](https://github.com/kubecost/docs/blob/2ea9021e8530369d53184ea5382b2e4c080bb426/allocation-api.md#aggregated-cost-model-api)
+       - `filter: <value>` -- configurable, accepts a single filter value
+       - `window: <N>d` or `<M>h` -- configurable, (1 ≤ N ≤ 7, 1 ≤ M ≤ 24)
+
+ 3. Spend Change -- sends an email and Slack alert reporting deviations in cost
```

Comment:
not really clear how this works or what this alert does based on this description...